### PR TITLE
#109: fixed.

### PR DESCRIPTION
**Pull request by PaulRambags**
This PR results from #102

### CODE REVIEW

File: src/org/daisy/dotify/formatter/impl/search/BlockAddress.java [hunk 1]
```diff
@@ -1,18 +1,24 @@
 package org.daisy.dotify.formatter.impl.search;
 
 public final class BlockAddress {
+    private final static Object NEXT_GROUP_NUMBER_SYNCHRONIZER = new Object();
```
Comment (bertfrees) [inline src/org/daisy/dotify/formatter/impl/search/BlockAddress.java]: Be careful with white space. This line has spaces while the other lines have tabs. Or was it done on purpose?

File: src/org/daisy/dotify/formatter/impl/search/BlockAddress.java [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
+
 	private static long nextGroupNumber = 0;
 	private final long groupNumber;
 	private final int blockNumber;
-
+
```
Comment (bertfrees) [inline src/org/daisy/dotify/formatter/impl/search/BlockAddress.java]: White space.

Comment (PaulRambags) [reply src/org/daisy/dotify/formatter/impl/search/BlockAddress.java]: White space fixed.

**Comment by PaulRambags** [comment]
Thanks @bertfrees - next time I will use better descriptive commit messages.

**Comment by kalaspuffar** [comment]
I'm running the regression tests.

**Comment by kalaspuffar** [comment]
Regression tests ended. 5 changed books. Will review the changes tomorrow. Good evening.

**Comment by bertfrees** [comment]
Funny. I hadn't expected this would have any effect. Good that you ran the tests.

**Comment by kalaspuffar** [comment]
Hi @PaulRambags and @bertfrees 

I've now reviewed the files and the changes to the files was empty pages that had been removed by this fix. And the rest seems correct so I'll merge this PR.

Best regards
Daniel

**Pull request merged**